.github/workflows/continuous-integration.yml

```
@@ -146,3 +146,37 @@ jobs:
 
       - name: Fetch Drupal cache
         run: yarn fetch-drupal-cache
+
+  build-e2e:
+    name: Build E2E
```

I think this could just be `build`. We originally had it as `build-e2e` to differentiate in CircleCI since it was specifically for running the E2E tests. But now we'll just have one build step in this workflow.

```suggestion
  build:
    name: Build
```